e2e/internal/e2e/apptainercmd.go

```diff
@@ -159,21 +159,6 @@ func ExpectOutput(mt MatchType, pattern string) ApptainerCmdResultOp {
 	}
 }
 
-// ExpectOutputf tests if the command output stream match the
-// formatted string pattern based on the type of match.
-func ExpectOutputf(mt MatchType, formatPattern string, a ...interface{}) ApptainerCmdResultOp {
```

Comment:
We should leave all of these unused functions in `e2e/internal/` because they are part of our `e2e` test toolkit and may be valuable in the future

Reply:
I prefer not to keep things around simply because `someone might use it`

Reply:
can we find a separate GOLANG e2e test framework and leave the maintenance to THAT community ?

Reply:
all of our e2e framework is made ourselves to make sure we can run tests and inspect containers effectively. I am not aware of an e2e test framework in golang that will allow us to do what we are currently doing in our e2e tests and dont think it is worth migrating to a different framework to avoid mainlining something that has been effective and stable to date

Reply:
reverting